src/ViewModels/CkanGroupViewModel.js

```diff
@@ -339,23 +374,31 @@ function populateGroupFromResults(viewModel, json) {
             // Extract the layer name from the WMS URL.
             var uri = new URI(wmsUrl);
```

Comment:
Actually here it is, since we should only be doing filterResultsByGetCapabilities for wms resources.